src/BlazorUI/Bit.BlazorUI/Components/Surfaces/Tooltip/BitTooltip.scss

```
@@ -4,10 +4,26 @@
     position: relative;
     display: inline-flex;
 
+    // One contract for every placement rule below, so a measurement is re-skinned - or handed over
+    // from C# as an inline custom property on this same element - in one place.
+    //
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)